CHANGELOG.md

```diff
@@ -64,7 +66,6 @@ CHANGELOG for Sulu
     * FEATURE     #820 [ContactBundle]   Contact-Import: define multiple tags: 'account_tag1 ..n'
     * FEATURE     #810 [ContactBundle]   Added command line tool for detecting missing country codes in import csv files
                                          that uses google geo api for finding the correct country code
-    * FEATURE     #805 [MediaBundle]     Implementing media move
```

Comment:
You didn't move it, you just deleted it. And are you sure it has not been released?